plenum/server/replica.py

```diff
@@ -1240,6 +1248,10 @@ def _apply_pre_prepare(self, pre_prepare: PrePrepare, sender: str) -> Optional[i
                                        valid_reqs,
                                        invalid_reqs,
                                        rejects)
+        duration = time.perf_counter() - start
```

Comment:
Why don't we put it into `processReqDuringBatch` to have `REQUEST_PROCESSING_TIME` be tracked in one place for both primary and backups?